### ISSUE TITLE
about CICDDoS2019 or CICDatasetCIC-IDS-2017 dataset

### ISSUE THREAD

**Issue by wshwhyu** [ahlashkari/CICFlowMeter#70]
Could you give me some suggestions about using machine learning CICDDoS2019 or CICDatasetCIC-IDS-2017  dataset? How to use the CICDDoS2019 or CICDatasetCIC-IDS-2017 data set to divide the test set and training set. Because there are too many data, how to effectively and quickly select the smaller test set and training set from  CICDDoS2019 or CICDatasetCIC-IDS-2017  dataset.Could you tell me how do you use  CICDDoS2019 or CICDatasetCIC-IDS-2017  dataset for machine learning to identify ddos?
I think the BoT-IoT dataset is a good dataset, but it uses the argus tool，not use  CICFlowMeter.The feature file csv division is very detailed as shown in the figure：
![image](https://user-images.githubusercontent.com/16811625/76499175-37e81b00-6479-11ea-8e44-dfd3aec19c01.png)

 I also want to select some data to do from ICDDoS2019 or CICDatasetCIC-IDS-2017  dataset . Could you give me some advices.thanks .

**Comment by ahlashkari** [comment]
Hi @wshwhyu,
As these are not related to this project and are about our datasets, pls contact me through email and I will reply there. (a.habibi.l@unb.ca)
